docs/UserGuide.md

```diff
@@ -484,16 +484,16 @@ Furthermore, certain edits can cause the MyBookshelf to behave in unexpected way
 1. Every library user starts with a default merit score of 0.
 1. **ONLY** library users with a **Merit Score >= Threshold** are allowed to borrow books.
 1. Library users can borrow multiple books as long as their merit score is above the threshold.
-1. The default threshold is set at -3. Use the `limit` command to set the threshold.
-1. `edit` can only be used to change the library user's personal information. This includes their name, phone number, email address, home address and tags. (Not merit score and borrowing book list).
-1. `add` and `edit` command **DOES NOT** support the direct adding and/or editing of merit score or user's book list.
-1. `add`, `delete`, `edit`, `clear` and `find` commands are for managing users, while `addbook`, `delbook`, `borrow`, `donate` and `return` commands are for managing books.
+1. The default threshold is set at -3. Use the [`limit`](#set-the-merit-score-threshold-of-the-library--limit) command to set the threshold.
+1. [`edit`](#editing-a-library-user--edit) can only be used to change the library user's personal information. This includes their name, phone number, email address, home address and tags. (Not merit score and borrowing book list).
+1. [`add`](#adding-a-library-user--add) and [`edit`](#editing-a-library-user--edit) command **DOES NOT** support the direct adding and/or editing of merit score or user's book list.
```

Comment:
Naming here is using user's book list instead of user's borrow book list.

Reply:
Fixed